### PR TITLE
chore(component): remove page footer component

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes https://github.com/unimelb/pattern-lib/issues/1684

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [x] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [x] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [x] Crossbrowser testing (Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

File: components/print.css [hunk 1]
```diff
@@ -10,8 +10,6 @@
   }
 
   .page-header,
```
Comment (jaredhenderson) [inline components/print.css:12]: Do you need to add some kind of class referencing the footer here, so the print styles still work for that area?

Comment (gepifanio) [reply components/print.css:13]: Fixed that

**Review by jaredhenderson — COMMENTED** [review]
Looking at the old footer in print version versus new:
### Old
Screen Shot 2021-07-14 at 11 09 17 am

### New
Screen Shot 2021-07-14 at 11 07 19 am

- Looks like the blank section on the left needs to be removed from print display as well so the content can use that space

**Comment by jaredhenderson** [comment]
One more thing - we are hiding some fairly important compliance information from printing now: 
Screen Shot 2021-07-14 at 11 36 22 am

Can these numbers be retained for print? The links on the left can remain hidden.

**Comment by gepifanio** [comment]
@jaredhenderson does like this looks fine?

image


That's what I could do to tidy up a bit the layout

**Review by jaredhenderson — APPROVED** [review]
Looks good. The print CSS doesn't have to be perfect, just needs to remove background colours and retain the important info, so this looks fine

**Pull request merged**